gcomm/src/gcomm/view.hpp

```diff
@@ -226,15 +227,16 @@ namespace gcomm
     class ViewState
     {
     public:
-        ViewState(UUID& my_uuid, View& view):
+        ViewState(UUID& my_uuid, View& view, gu::Config& conf):
                 my_uuid_(my_uuid),
-                view_(view)
+                view_(view),
+                file_name_(get_viewstate_file_name(conf))
         { }
         std::ostream& write_stream(std::ostream& os) const;
         std::istream& read_stream(std::istream& is);
-        void write_file(const char* fname = NULL) const;
-        bool read_file(const char* fname = NULL);
-        static void remove_file(const char* fname = NULL);
+        void write_file() const;
+        bool read_file();
+        static void remove_file(gu::Config& conf);
```

Comment:
Hm, why is this static? I know that it WAS static, but why? Do we need to use that method somewhere we don't have a ViewState object?